### ISSUE TITLE
"java.lang.NoSuchMethodError: scala.Predef$.refArrayOps" while running ScalaCheck on 2.11.12

### ISSUE THREAD

**Issue by eed3si9n** [sbt/sbt#4608]
## steps

1. build sbt locally based on b40084f6fdab5b8dc7d7534ddd843fba348e0893
2. git clone https://github.com/Philippus/bump.git && cd bump && git checkout 0574b88d34a92ea3265105415003baca2140cbb4
3. sbt -Dsbt.version=1.3.0-LOCAL-b40084f "++2.11.12\!" test

## problem

```
[error] java.lang.NoSuchMethodError: scala.Predef$.refArrayOps([Ljava/lang/Object;)Lscala/collection/mutable/ArrayOps;
[error] 	at org.scalacheck.util.CmdLineParser$class.parseArgs(CmdLineParser.scala:81)
[error] 	at org.scalacheck.Test$$anon$1.parseArgs(Test.scala:228)
[error] 	at org.scalacheck.Test$$anon$1.parseParams(Test.scala:276)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[error] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error] 	at java.lang.reflect.Method.invoke(Method.java:498)
[error] 	at org.scalacheck.ScalaCheckFramework$$anon$4.<init>(ScalaCheckFramework.scala:196)
[error] 	at org.scalacheck.ScalaCheckFramework.runner(ScalaCheckFramework.scala:191)
[error] 	at sbt.Defaults$.$anonfun$createTestRunners$1(Defaults.scala:979)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:237)
[error] 	at scala.collection.immutable.Map$Map1.foreach(Map.scala:128)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:237)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:230)
[error] 	at scala.collection.AbstractTraversable.map(Traversable.scala:108)
[error] 	at sbt.Defaults$.createTestRunners(Defaults.scala:973)
[error] 	at sbt.Defaults$.allTestGroupsTask(Defaults.scala:1041)
[error] 	at sbt.Defaults$.$anonfun$testTasks$10(Defaults.scala:784)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:268)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.Execute.work(Execute.scala:277)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:268)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] (Test / executeTests) java.lang.NoSuchMethodError: scala.Predef$.refArrayOps([Ljava/lang/Object;)Lscala/collection/mutable/ArrayOps;
```

## expectation

Test runs.

**Comment by eatkins** [comment]
Whoops. Should be fixed in #4610.

**Comment by eed3si9n** [comment]
That was a quick turnaround!
